### DISCUSSION TITLE
Installed Brew on M1 Mac - Problem !

### DISCUSSION THREAD

**Discussion by Zebbyz** [Homebrew/discussions#1417] [Getting started]
Got this at the end  Warning: /opt/homebrew/bin is not in your PATH.

Wants me to do this but just not sure about exactly what to do

==> Next steps:
- Add Homebrew to your PATH in /Users/guy/.zprofile:
    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/guy/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"

Just complete newb and only trying to get pdftohtml working on my system if anyone can assist that would help a lot thanks

**Comment by carlocab** [comment]
Did you try running
```sh
echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/guy/.zprofile
eval "$(/opt/homebrew/bin/brew shellenv)"
```
?

**Reply by Zebbyz** [reply]
I did run the echo line and it returned the eval line below it... (didn't know what to do with the >> /users... part of it tho

**Reply by Zebbyz** [reply]
Just completely unsure of the context and sequence of code... what's a reference and what's the line to enter

**Reply by carlocab** [reply]
You can Google "Unix PATH" and "shell eval" for references. You need to run the full command and not leave out parts of it for it to do what's needed. The `>>` appends the output of the `echo` command (which you saw when you did it on its own) to the file at `/Users/guy/.zprofile`.

**Reply by Zebbyz** [reply]
Ah ok thanks heaps

echo 'eval "$(/opt/homebrew/bin/brew/Users/guy/.zprofile shellenv)"'
eval "$(/opt/homebrew/bin/brew shellenv)"

ran this and SUCCESS

Then ran
brew install pdftohtml
and SUCCESS

Many thanks... for the confidence !
g

**Comment by Zebbyz** [comment]
Next problem is that brew has build pdftohtml in this location

opt/homebrew/bin/pdftohtml

But the software that's using it can't find it... any ideas on how to move the install location - I don't mind reinstalling it just need to point it

**Reply by SMillerDev** [reply]
That's a software bug, it should always look in PATH

**Reply by Zebbyz** [reply]
ok thanks ill check

**Comment by ahmedhossam91** [comment]
is there any way to install brew offline ?

**Reply by gromgit** [reply]
You can pull a tarball of the Homebrew software itself, then install it offline: https://docs.brew.sh/Installation#untar-anywhere

However, you need to be online to download the Homebrew software database (and of course the formulae you want to install).

**Comment by ahmedhossam91** [comment]
My problem that my mac os virsion majova installed on dell laptop and I can’t identify the network so of u could help me I would be thankful

**Reply by gromgit** [reply]
As you're running a Hackintosh, I suggest asking the person whose installation instructions you followed.
